.circleci/config.yml

```diff
@@ -40,6 +40,31 @@ jobs:
       - run:
           command: cargo audit
 
+  load-test-checks:
+    docker:
+      - image: cimg/python:3.11
+    working_directory: "~/autopush-rs/tests/load"
+    steps:
+      - checkout:
+          path: ~/autopush-rs/
+      - run:
```

Comment:
Not critical but these steps could live in the top level Makefile instead

Reply:
I didn't see that Makefile! I will move them!

Reply:
@pjenvey Make commands added, can you please check them, MakeFiles are not my specialty.